scripts/vif-real

```diff
@@ -227,8 +240,11 @@ remove_from_bridge()
         # port.  Use --if-exists to suppress the error that would otherwise
         # arise in that case.
         $vsctl --timeout=30 -- --if-exists del-port $dev
+        xenstore-rm ${PRIVATE}/"$TYPE"-ofport
```

Comment:
On a second thought, I added xenstore-rm here, to remove tap-ofport and tap-ifindex (once pv driver kicks in). As this is shared between vif/tap, tap-* will linger till the whole path is removed.